packages/ui/src/ActionDialog/ActionDialog.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { ComponentMeta } from "@storybook/react";
+
+import ActionDialog from "./ActionDialog";
+
+export default {
+  title: "Action Dialog",
+  component: ActionDialog,
+} as ComponentMeta<typeof ActionDialog>;
+
+export const Default = (): JSX.Element => (
+  <div className="w-screen h-screen bg-gray-800">
```

Comment:
Sorry about giving an opinion just now, but I've thought about what you mentioned (adding overlay). I think we can include the overlay in the component itself and have a click outside the modal body call to `onCancel`, thus closing the modal.
For this purpose I've copied the `useClickOutside` hook over to `ui` package and it can be used for this purpose.
For usage/reference, check [this](https://github.com/eisbuk/EisBuk/blob/master/packages/client/src/react-redux-firebase-auth/ui/atoms/AuthErrorDialog.tsx)

Reply:
I think I made this comment before learning that the Dialog controller should only render one Dialog in a fixed position at a time. Is it better for the controller to define the overlay and use the `useClickOutside` hook, rather than each Dialog layering an overlay on top of the others?

Reply:
Yeah...ok, it makes sense to make it just a stateless component. However, as such, it shouldn't care if it's a modal or not, so it can simply take in `onCancel` and `onConfirm` (can just be passed as click handlers) and the controller will decide if it wants to use it as modal or not, implement the concept of open/closed, etc.